apps/src/types/blocklyTypes.ts

```diff
@@ -0,0 +1,20 @@
+export interface BlockDefinition {
```

Comment:
Good question. I think so far we've put types mostly next to the code they are relevant to (i.e. music types unser `/music` and lab2 types under `/lab2`. The `types` folder currently has `progressTypes`, which are generic enough to make some sense to be here. To keep with the current pattern I would maybe put this file under `/blockly` instead. I don't think we will want all type definitions under `/types` because our repo is so big, but I'm open to discussion on that!

Reply:
Thanks @molly-moen - SGTM! I'll move and rename file as `/blockly/types.ts`.